### ISSUE TITLE
[NTSC] F-Zero GX freezes on memory card screen

### ISSUE THREAD

**Issue by ploder424** [FIX94/Nintendont#185]
I'm using Nintendont downloaded from here on 12th December 2015. Loading NTSC F-Zero GX for first time via USB Loader GX with full nand memory card emulation. It tries to make a new save file but then freezes on this screen. No further input with official gamecube controller is recognised. I have to power off wii by holding power button.  Restarting game leads to the same freeze on the memory card screen.

**Comment by carnage702** [comment]
put all your nintendont settings here, also maybe try using nintendont directly from homebrew channel and not trough usbloader gx.

**Comment by ploder424** [comment]
Ok, I will try when I have access to my wii at the weekend.

**Comment by SevenWillis** [comment]
Make sure the wiird debug is off and turn off the log and OSreport if it isn't already. Also, make sure that the memory card blocks aren't set above 251 as it's unstable

**Comment by ploder424** [comment]
[Nintendont config.zip](https://github.com/FIX94/Nintendont/files/67415/Nintendont.config.zip)
I tried loading directly through Homebrew and it progresses past the mem card screen, but the whole screen was red tinted. 

I can see from settings that Nintendont v.3.381 was set to NTSC video mode. This seems to be a video mode issue. I tried setting Nintedont back to PAL60 and it gets stuck on the same screen. I usually have USB loader GX set to force PAL60 so that seems to be the problem. 

I'm using an NTSC version of the game. Hash checksum matches that on Redump.org database. I don't usually have an issue with forcing PAL60 on NTSC games. It is usually first party titles like Luigi's Mansion, Mario tennis etc. that give me problems, but they usually fail to load at all. 

Wiird debug, log and OSreport are all off. Mem card blocks aren't above 251. AFAIK all settings are default.
